api/android/README.md

```diff
@@ -43,44 +43,53 @@ export GSTREAMER_ROOT_ANDROID=$ANDROID_DEV_ROOT/gstreamer-1.0
 export NNSTREAMER_ROOT=$ANDROID_DEV_ROOT/workspace/nnstreamer
 ```
 
-#### Download Android Studio
+### Install required packages
+Some required packages should be installed as below.
+```bash
+$ sudo apt install subversion
```

Comment:
Add `curl`

Reply:
To whom are you speaking? Please add them :P

Reply:
It's for me not to forget it. Because I tested it at home.